runtime/JavaScript/src/antlr4/CharStream.js

```diff
@@ -4,8 +4,6 @@
  */
 
 import Token from './Token.js';
-import './polyfills/codepointat.js';
```

Comment:
how does this relate to the PR ?

Reply:
Those polyfills are modifying the String.prototype if you look at the code.

For example here:

https://github.com/antlr/antlr4/blob/master/runtime/JavaScript/src/antlr4/polyfills/codepointat.js#L58

I did check that these polyfills are no longer needed, we've come a long way since they were added around 6 years ago! Now those functions that were polyfilled are available everywhere.

https://caniuse.com/mdn-javascript_builtins_string_codepointat
https://caniuse.com/mdn-javascript_builtins_string_fromcodepoint